inductiva/_cli/cmd_tasks/list.py

```diff
@@ -14,40 +17,40 @@ def list_tasks(args):
     """
     if args.task_id is not None:
         task_list = [tasks.Task(args.task_id)]
-
     else:
         last_n = 5 if args.last_n is None else args.last_n
         task_list = tasks.get(last_n=last_n)
 
     if not task_list:
-        print("No tasks found.")
+        print("No tasks found.", file=fout)
         return 1
 
     table_dict = tasks.to_dict(task_list)
 
-    emph_formatter = utils.format_utils.get_ansi_formatter()
+    emph_formatter = format_utils.get_ansi_formatter()
 
     def color_formater(status):
         if status == models.TaskStatusCode.SUCCESS:
-            return emph_formatter(status, utils.format_utils.Emphasis.GREEN)
+            return emph_formatter(status, format_utils.Emphasis.GREEN)
         elif status in tasks.Task.FAILED_STATUSES:
-            return emph_formatter(status, utils.format_utils.Emphasis.RED)
+            return emph_formatter(status, format_utils.Emphasis.RED)
         return status
 
     formatters = {
-        "Submitted": [utils.format_utils.datetime_formatter,],
-        "Started": [utils.format_utils.datetime_formatter,],
+        "Submitted": [format_utils.datetime_formatter,],
+        "Started": [format_utils.datetime_formatter,],
         "Status": [color_formater]
     }
 
     header_formatters = [
-        lambda x: emph_formatter(x.upper(), utils.format_utils.Emphasis.BOLD)
+        lambda x: emph_formatter(x.upper(), format_utils.Emphasis.BOLD)
     ]
 
-    print(
-        utils.format_utils.get_tabular_str(table_dict,
-                                           formatters=formatters,
-                                           header_formatters=header_formatters))
+    print(format_utils.get_tabular_str(table_dict,
```

Comment:
this print is adding a blank line at the end that prevents the paginator from being able to scroll to the last line.
Well, not really, because it is indeed scrolling but the last line is an empty line so it gives the idea that the content of the file went away.